src/pages/blog/status-check-monitoring-with-playwright-on-checkly.mdx

```diff
@@ -0,0 +1,66 @@
+}
+
+## Introduction
+
+Having a safe, functional, and feature-rich configured monitoring and status checks in place is nowadays needed to ensure a fast error reporting and by that a good uptime so your SLA is getting respected. The typical monitoring solutions are e.g. Grafana which would be feature-rich but has no good native integration for launching browsers to emulate a real user. Also, you need to host it yourself which is mostly for monitoring solutions not advised (or you need a HA installation in another data center or cloud provider).
+
+To fix these issues, Checkly has created a product to provide developer-friendly status checks, which can be used to launch browsers in the cloud. By using it you also don't need to spin up an own instance in your infrastructure since Checkly is a SaaS (Software as a Service) product and they host it for you. They even provide a full public dashboard for your customers. Starting from today they provide a full Playwright integration.
+
+<img src="/blog/status-check-monitoring-with-playwright-on-checkly/checkly-home-dashboard.png" style={{maxWidth: "80%"}}/>
+
+## Usage
+
+There are two types of checks which you can use. API Checks to validate a given response body, status code, or schemas like JSON or GraphQL or browser-based tests on which we'll focus on this blog post. Browser-based tests are running a headless Chromium instance in the cloud to validate e.g. that your production login is working. Other common alternative scenarios would be:
+
+- Buy an item in your shop and validate that the user flow is working.
+- Validate that the OAuth login with e.g. Google is working.
+- Using a search on your website to ensure your elastic search in the background is correctly configured and serves the expected data.
+
+Checkly itself gives you the ability to run any Node.js related code either with Playwright or Puppeteer to automate your user flows. They have a secret store built in to provide the secrets over to your code once it is running.
+
+```js
+const assert = require("chai").assert;
+const playwright = require("playwright");
+
+const browser = await playwright.chromium.launch();
+const page = await browser.newPage();
+await page.goto("https://playwright.tech");
+
+assert.equal(await page.title(), "Everything about the Playwright framework - Playwright Community 🎭");
+await page.waitForSelector("text=Playwright Community");
+await page.waitForSelector("text=Try Playwright");
+assert.isAbove(await page.$$eval("article", elements => elements.length), 8)
+await browser.close();
+```
+
+In this example which we're ensuring the following checks:
+
+- Assert that the title has the expected value
+- Assert that "Playwright Community" is on the page
+- Assert that "Try Playwright" is on the page
+- Assert that at least 8 articles are on the page
+
+If one of them is failing, Checkly will automatically treat this as a failing test and alert the user by e.g. Slack, PagerDuty, SMS, or Email.
+
+## Integrations
```

Comment:
Suggestion: mention Terraform. It is one of the most used integrations and allows (through the Checkly Terraform Provider) to automatically manage large numbers of checks - a gamechanger when large APIs and hundreds of checks are involved. https://github.com/checkly/terraform-provider-checkly